src/Tribe/Event_Cleaner_Scheduler.php

```diff
@@ -150,7 +150,14 @@ public function select_events_to_purge( $month ) {
 		INNER JOIN {$wpdb->postmeta} AS t2 ON t1.ID = t2.post_id
 		WHERE t1.post_type = %d
 			AND t2.meta_key = '_EventEndDate'
-			AND t2.meta_value <= DATE_SUB( CURDATE(), INTERVAL %d MONTH )";
+			AND t2.meta_value <= DATE_SUB( CURDATE(), INTERVAL %d MONTH )
+			AND t1.post_parent = 0
+			AND t1.ID NOT IN (
+				SELECT DISTINCT post_parent
```

Comment:
Maybe it would be worth saving this on a another SQL variable to make is more readable?
`AND t1.ID NOT IN ( $posts_with_parents_sql )`